tests/scanbuild_py_makefile.txt

```diff
@@ -0,0 +1,12 @@
+# Test scan-build-py
+# REQUIRES: scanbuildpy
+
+# RUN: rm -rf %t
+# RUN: mkdir -p %t
+# RUN: cat %s > %t/Makefile
+# RUN: cp %S/scanbuild_py.cpp %t
+# RUN: cd %t && %scanbuildpy make|grep "to examine bug reports"
+# RUN: cd %t && %scanbuildpy --intercept-first make && test -f compile_commands.json
```

Comment:
Should we make the dependency of `make` part of the requires? I think so...

Reply:
do you? I think make is installed all the time?!